bridge/client/app/_components/ktb-variable-selector/ktb-variable-selector.component.html

```diff
@@ -0,0 +1,11 @@
+<button
+  dt-icon-button
+  variant="nested"
+  aria-label="Add secret"
+  ktbTreeListSelect
+  [data]="data"
+  [options]="options"
+  (selected)="setVariable($event)"
+>
+  <dt-icon name="resetpassword"></dt-icon>
```

Comment:
icon name should be configurable, as we will set other things than secrets with it